package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lfpwebapp",
+  "version": "0.0.1",
+  "description": "This is a skeleton for a node.js + express.ts app.",
+  "main": "dist/server.js",
+  "scripts": {
+    "start": "npm run build && node public/server.js",
+    "build": "touch public && rm -dr public && tsc && cp -r src/views src/assets public/",
+    "devstart": "nodemon --config nodemon.json",
+    "tslint": "./node_modules/.bin/tslint -c tslint.json -p tsconfig.json",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "dependencies": {
+    "@types/cookie-parser": "^1.4.2",
+    "body-parser": "^1.18.2",
+    "cookie-parser": "^1.4.4",
+    "dotenv": "^8.2.0",
+    "express": "^4.15.5",
+    "express-session": "^1.15.6",
+    "express-ts": "0.0.1-dev.5",
```

Comment:
I failed to find what it is. Not maintained for 3 years. Please remove unless really, really needed.